tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+log_format = [%(asctime)s.%(msecs)03d %(levelname)s] %(message)s
+log_date_format = %Y-%m-%d %H:%M:%S
+log_file_level=INFO
```

Comment:
What does it mean that we only specify INFO here?
Can we still us e.g logging.error(...) in the code?

Reply:
`logging` provides several log "levels" (an integer greater than 0), see the list [here](https://docs.python.org/3/library/logging.html#levels). For some reason, I thought that INFO was the lowest level, but I now see that it is DEBUG.

The effect of this is that any log done with level less-than INFO (so level 19 or lower) will not be printed in case of errors. So this means that logging INFO, WARNING, ERROR and CRITICAL will all test fail, but DEBUG isn't. 

Should DEBUG be printed as well?